modules/kubernetes/config_schema.json

```diff
@@ -22,8 +22,8 @@
   },
   "required": [
     "host",
-    "ca_cert",
-    "client_cert",
-    "client_key"
+    "ca_data",
+    "cert_data",
+    "key_data"
```

Comment:
whats the reason for changing these keys?

Reply:
to keep it same as the Kubernetes config names (previously, it was not 1:1 mapping which might cause confusion)